### PR TITLE
Dev Tools week 3 day 7 update

### PR DESCRIPTION
**Pull request by Jaylyn-Barbee**
---

⚠️ Merge/Release date 3/6

---

### CODE REVIEW

File: docs/30DaysOfPWA/dev-tools/06.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # 3.6: Test automation tools
 
-## *Coming soon!*
+**Author: Jaylyn Barbee [@jaylynsatwork](https://twitter.com/jaylynsatwork)**
+
+Welcome to **Week 3 Day 1** of #30DaysOfPWA! Want to learn more about this project? Check out our [Kickoff](../kickoff.md) post to get more details on the content roadmap and contributors. Now, let's dive in!
```
Comment (zateutsch) [inline docs/30DaysOfPWA/dev-tools/06.md:5]:
```suggestion
Welcome to **Week 3 Day 7** of #30DaysOfPWA! Want to learn more about this project? Check out our [Kickoff](../kickoff.md) post to get more details on the content roadmap and contributors. Now, let's dive in!
```

File: docs/30DaysOfPWA/dev-tools/07.md [hunk 2]
```diff
@@ -15,3 +15,59 @@ Welcome to **Week 3 Day 7** of #30DaysOfPWA! Want to learn more about this proje
 | **Introduction** | Introduction to packaging for the Windows Store |
 | **The Process** | Overview of how to package for the actual process of using pwabuilder.com to package for the Windows Store |
 | **Why the Microsoft Store?** |An explanation of why you would chose the Microsoft Store over any others |
+| **Conclusion** | Wrapping up |
```
Comment (zateutsch) [inline docs/30DaysOfPWA/dev-tools/07.md:18]:
```suggestion
```

File: docs/30DaysOfPWA/dev-tools/07.md [hunk 3]
```diff
@@ -15,3 +15,59 @@ Welcome to **Week 3 Day 7** of #30DaysOfPWA! Want to learn more about this proje
...
+
+## The Process
+
+### Step 1: Packaging
+In order to package for the Windows store, you are going to want to navigate to pwabuilder.com and enter the url to your web app on the home page. 
+
+![pwa builder home page](/docs/30DaysOfPWA/dev-tools/_media/07_pwabuilder_home.png)
+
+ 
+After a few tests run to verify the quality of your PWA (which we all know is out of this world thanks to PWA Studio) you’ll receive a report card detailing what you do well and some areas for improvement.
```
Comment (zateutsch) [inline docs/30DaysOfPWA/dev-tools/07.md:46]:
```suggestion
After a few tests run to verify the quality of your PWA, you’ll receive a report card detailing what you do well and some areas for improvement.
```

Comment (zateutsch) [reply docs/30DaysOfPWA/dev-tools/07.md:46]: Trying to stay away from advertisement-ey promo

File: docs/30DaysOfPWA/dev-tools/07.md [hunk 4]
```diff
@@ -15,3 +15,59 @@ Welcome to **Week 3 Day 7** of #30DaysOfPWA! Want to learn more about this proje
...
+
+![pwa builder report card page](/docs/30DaysOfPWA/dev-tools/_media/07_pwabuilder_rc.png)
+ 
+At the bottom of this page, you can click next and press “Next” and you will be at the publishing page of PWABuilder. We can press “Store Package” in the Windows section and we are greeted with a pop up to input a few options.
+
+![pwa builder publish page](/docs/30DaysOfPWA/dev-tools/_media/07_pwabuilder_pub.png)
+   
+In a new tab, you are going to want to head over to Partner Center and register your app. 
+**It is important that you register your app on Partner Center with the same name you listed in your manifest. When it comes time to upload your app bundles, it will check if the bundle matches the name of the registered app.**
+Information on how to register your app can be found here and information on where to find the necessary Ids can be found here. Once you have inputted your publisher information, you can press generate and download your app bundles.
```
Comment (zateutsch) [inline docs/30DaysOfPWA/dev-tools/07.md:56]: Are you missing links here?

File: docs/30DaysOfPWA/dev-tools/07.md [hunk 5]
```diff
@@ -15,3 +15,59 @@ Welcome to **Week 3 Day 7** of #30DaysOfPWA! Want to learn more about this proje
...
+### Step 2: Partner Center
+If you are inside of your app on Partner Center, you will see a button that says, “Start your submission”. After clicking on that you will see a list of options you will need to take care of before your submission is ready to be reviewed. Since you made sure to have everything on the checklist above, these sections should be a breeze. 
+
+![partner center submission page](/docs/30DaysOfPWA/dev-tools/_media/07_pc.png)
+ 
+After completing each task under the headers colored blue, your app is ready for review. The review process takes about 24 hours and the feedback will either be precisely what was wrong with your submission so you can fix it or a congratulations message letting your know that your submission has been processed and your app will be in the store within the next two hours.
+ 
+![windows store webboard product description page](/docs/30DaysOfPWA/dev-tools/_media/07_store.png)
+
+The PWA Builder process is designed to create and produce store ready PWAs. If you are using the tools at your disposal, the app submission process will be simple, and your product description page will be robust.
```
Comment (zateutsch) [inline docs/30DaysOfPWA/dev-tools/07.md:67]:
```suggestion
PWA Builder helps you to create store ready PWAs with robust descriptions pages: all while keeping the app submission process as simple as possible.
```

File: docs/30DaysOfPWA/dev-tools/07.md [hunk 6]
```diff
@@ -15,3 +15,59 @@ Welcome to **Week 3 Day 7** of #30DaysOfPWA! Want to learn more about this proje
...
+ 
+![windows store webboard product description page](/docs/30DaysOfPWA/dev-tools/_media/07_store.png)
+
+The PWA Builder process is designed to create and produce store ready PWAs. If you are using the tools at your disposal, the app submission process will be simple, and your product description page will be robust. 
+
+## Why the Microsoft Store?
+The Microsoft store is an excellent place for your app to live. All of your in-app purchases (Whether that be through Stripe or a custom payment system) go 100% to you. Microsoft will not take a cut. All of your analytics that you have running for your website will seemlessly. And most importantly, PWAs submitted through the Microsoft Store are TRUE PWAs; meaning they are not wrapped in a webview. There will be no "gotchas" or extra hoops to jump through in order for your app to feel native. It will just work. I also promise I am not making this up, I submitted my very own app, "Sync'd - a Social Calendar App", to the Microsoft Store in early February. I downloaded the app and it look flawless, exactly like I expected. The cherry on top is that my app was reviewed, approved and in the store in 24 hours.
```
Comment (zateutsch) [inline docs/30DaysOfPWA/dev-tools/07.md:70]:
```suggestion
The Microsoft store is an excellent place for your app to live. All of your in-app purchases (Whether that be through Stripe or a custom payment system) go 100% to you. Microsoft will not take a cut. All of your analytics that you have running for your website will behave seamlessly. And most importantly, PWAs submitted through the Microsoft Store are TRUE PWAs; meaning they are not wrapped in a webview. There will be no "gotchas" or extra hoops to jump through in order for your app to feel native. It will just work. I also promise I am not making this up, I submitted my very own app, "Sync'd - a Social Calendar App", to the Microsoft Store in early February. I downloaded the app and it look flawless, exactly like I expected. The cherry on top is that my app was reviewed, approved and in the store in 24 hours.
```

Comment (zateutsch) [reply docs/30DaysOfPWA/dev-tools/07.md:70]: We might be pressing our foot a bit hard on the gas here with the Microsoft store.

Instead of making promises, we should just present what the store has to offer and let it stand on its own. Any sort of guarantee or testimonial seems hollow when coming from an msoft employee.

File: docs/30DaysOfPWA/dev-tools/07.md [hunk 7]
```diff
@@ -15,3 +15,59 @@ Welcome to **Week 3 Day 7** of #30DaysOfPWA! Want to learn more about this proje
...
+
+## Conclusion
```
Comment (zateutsch) [inline docs/30DaysOfPWA/dev-tools/07.md:72]:
```suggestion
## Wrapping Up
```

File: docs/30DaysOfPWA/dev-tools/07.md [hunk 8]
```diff
@@ -15,3 +15,59 @@ Welcome to **Week 3 Day 7** of #30DaysOfPWA! Want to learn more about this proje
...
+PWABuilder makes packaging for the Microsoft Store easier than ever. For additional resources on how to package and much more, you can visit the [PWABuilder Blog](https://blog.pwabuilder.com/) page. It is also important to mention you can also package your app for the Google Play and Apple App Stores however, your PWA will be hindered slightly in its capabilities.
```
Comment (zateutsch) [inline docs/30DaysOfPWA/dev-tools/07.md:73]: resources and links section?

**Review by zateutsch — CHANGES_REQUESTED** [review]
Minor feedback except for the "Why The Microsoft Store?" section, which I think may need some work to avoid coming off as a conflict of interest.

Thanks, Jaylyn! I think this is almost there

File: docs/30DaysOfPWA/dev-tools/06.md (hunk 1 above)
Comment (captainbrosset) [reply docs/30DaysOfPWA/dev-tools/06.md:5]: Actually this article is named `06.md`, so the right change is probably "Week 3 Day 6". But why is this change being made in this PR at all? @nitya is already updating this article in PR #72. So better revert this change here.

File: docs/30DaysOfPWA/dev-tools/07.md [hunk 9]
```diff
@@ -15,3 +15,59 @@ Welcome to **Week 3 Day 7** of #30DaysOfPWA! Want to learn more about this proje
 | **Introduction** | Introduction to packaging for the Windows Store |
 | **The Process** | Overview of how to package for the actual process of using pwabuilder.com to package for the Windows Store |
 | **Why the Microsoft Store?** |An explanation of why you would chose the Microsoft Store over any others |
+| **Conclusion** | Wrapping up |
+
+# Packaging for the Windows Store
+
+## Introduction
+Great! You’ve just finished your web app, you’ve used the PWA Studio VS Code extension to enhance your application into a quality PWA. You can use Azure Static Web Apps to host your site, but if you’ve already taken care of that you are ready to send it off into the stores! Luckily for you, PWA Builder makes that as easy as pie. In this paper we will cover how to use our tools to package your PWA for the Microsoft Store. This is the last step, sharing your PWA with the world and now its time for all your hard work to pay off! Before we get started, here are some things you might want to have ready:
```
Comment (captainbrosset) [inline docs/30DaysOfPWA/dev-tools/07.md:23]:
```suggestion
Great! You’ve just finished your web app, you’ve used the PWA Studio VS Code extension to enhance your application into a quality PWA. You can use Azure Static Web Apps to host your site, but if you’ve already taken care of that you are ready to send it off into the stores! Luckily for you, PWA Builder makes that as easy as pie. In this article we will cover how to use our tools to package your PWA for the Microsoft Store. This is the last step, sharing your PWA with the world and now its time for all your hard work to pay off! Before we get started, here are some things you might want to have ready:
```

Also, this article comes at the end of week 3, and at this point, readers who've followed the series have used the PWA Studio extension, as you mention, but not only. They've also used debugging, auditing tools, testing tools, and more. So it feels like this intro is ignoring them. Could you please rephrase a bit? Either mention them all, or none at all and just go with something like "Great! You're web app is ready, and you can now send it off into the stores!"

File: docs/30DaysOfPWA/dev-tools/07.md [hunk 10]
```diff
@@ -15,3 +15,59 @@ Welcome to **Week 3 Day 7** of #30DaysOfPWA! Want to learn more about this proje
...
+
+- 512 x 512 Icon (at least)
```
Comment (captainbrosset) [inline docs/30DaysOfPWA/dev-tools/07.md:25]:
```suggestion
- A 512px by 512px icon (at least)
```

File: docs/30DaysOfPWA/dev-tools/07.md [hunk 11]
```diff
@@ -15,3 +15,59 @@ Welcome to **Week 3 Day 7** of #30DaysOfPWA! Want to learn more about this proje
...
+    - This will be used for your app icon
+- Screenshots
+    - For the Windows platform, you’re going to want laptop/tablet sized screenshots of your app to display in a carousel on your product description page. They have pixel recommendations in partner center under your store listings.
```
Comment (captainbrosset) [inline docs/30DaysOfPWA/dev-tools/07.md:28]: By "Windows platform", do you mean the Microsoft Store? If so, maybe switch it to that.
Also, you're mentioning the partner center here, but I don't think readers of the blog series have been exposed to the partner center yet in earlier articles. Maybe rephrase to "The exact pixel dimensions will be given to you in the Partner Center, where you will be publishing your app for the Microsoft Store (more on that later)".

File: docs/30DaysOfPWA/dev-tools/07.md [hunk 12]
```diff
@@ -15,3 +15,59 @@ Welcome to **Week 3 Day 7** of #30DaysOfPWA! Want to learn more about this proje
...
+- Short Description & Long Description
+    - The short description may be the same as what you have in your manifest while the long description is a more in-depth summary of what your app is and what it does
+- A privacy policy if your app can access, collect, or transmit personal information (data that could be used to identify a person).
+- A personal Microsoft account
```
Comment (captainbrosset) [inline docs/30DaysOfPWA/dev-tools/07.md:32]: Could you link to where someone would create an account?

File: docs/30DaysOfPWA/dev-tools/07.md (hunk 9 above)
Comment (captainbrosset) [reply docs/30DaysOfPWA/dev-tools/07.md:23]: A second comment about that: in this intro we talk about stores (plural), which is great. But then immediately after we start being very specific about the Microsoft Store.

It might be great to add a section in between that says something like "PWABuilder makes it possible to submit your app to multiple different stores (`<list which ones here>`), but in this article we will focus on how to submit to the Microsoft Store, via the Partner Center".

File: docs/30DaysOfPWA/dev-tools/07.md [hunk 13]
```diff
@@ -15,3 +15,59 @@ Welcome to **Week 3 Day 7** of #30DaysOfPWA! Want to learn more about this proje
...
+- $20 to pay for your Partner Center account
```
Comment (captainbrosset) [inline docs/30DaysOfPWA/dev-tools/07.md:33]: Maybe add a word about why you have to pay 20 bucks? And whether other stores also charge you money?

File: docs/30DaysOfPWA/dev-tools/07.md [hunk 14]
```diff
@@ -15,3 +15,59 @@ Welcome to **Week 3 Day 7** of #30DaysOfPWA! Want to learn more about this proje
...
+- The URL to your site.
+
+With the above four items, you are ready to begin.  
+
+## The Process
```
Comment (captainbrosset) [inline docs/30DaysOfPWA/dev-tools/07.md:38]: nit: I don't think this h2-level header adds much value. I suggest removing it and raising the step 1 and step 2 headers to h2, instead of h3 levels.

File: docs/30DaysOfPWA/dev-tools/07.md [hunk 15]
```diff
@@ -15,3 +15,59 @@ Welcome to **Week 3 Day 7** of #30DaysOfPWA! Want to learn more about this proje
...
+
+### Step 1: Packaging
+In order to package for the Windows store, you are going to want to navigate to pwabuilder.com and enter the url to your web app on the home page.
```
Comment (captainbrosset) [inline docs/30DaysOfPWA/dev-tools/07.md:41]:
```suggestion
In order to package for the Microsoft store, you are going to want to navigate to [pwabuilder.com](https://pwabuilder.com) and enter the URL to your web app on the home page. 
```

File: docs/30DaysOfPWA/dev-tools/07.md [hunk 16]
```diff
@@ -15,3 +15,59 @@ Welcome to **Week 3 Day 7** of #30DaysOfPWA! Want to learn more about this proje
...
+In order to package for the Windows store, you are going to want to navigate to pwabuilder.com and enter the url to your web app on the home page. 
+
+![pwa builder home page](/docs/30DaysOfPWA/dev-tools/_media/07_pwabuilder_home.png)
+
+ 
+After a few tests run to verify the quality of your PWA (which we all know is out of this world thanks to PWA Studio) you’ll receive a report card detailing what you do well and some areas for improvement.
+
+![pwa builder report card page](/docs/30DaysOfPWA/dev-tools/_media/07_pwabuilder_rc.png)
+ 
+At the bottom of this page, you can click next and press “Next” and you will be at the publishing page of PWABuilder. We can press “Store Package” in the Windows section and we are greeted with a pop up to input a few options.
```
Comment (captainbrosset) [inline docs/30DaysOfPWA/dev-tools/07.md:50]: > click next and press “Next”

Seems like one of these 2 needs to be removed 😉

Comment (captainbrosset) [reply docs/30DaysOfPWA/dev-tools/07.md:50]: The first sentence here uses "you", and the second uses "we". Let's stay consistent throughout. I think you've used "you" mostly in this article, which is fine. So based on that, rephrase the second sentence to "You can press Store Package in the Windows section and you will then be greeted with a pop up to enter a few options".

File: docs/30DaysOfPWA/dev-tools/07.md [hunk 17]
```diff
@@ -15,3 +15,59 @@ Welcome to **Week 3 Day 7** of #30DaysOfPWA! Want to learn more about this proje
...
+
+![pwa builder publish page](/docs/30DaysOfPWA/dev-tools/_media/07_pwabuilder_pub.png)
+   
+In a new tab, you are going to want to head over to Partner Center and register your app.
```
Comment (captainbrosset) [inline docs/30DaysOfPWA/dev-tools/07.md:54]: Please link to Partner Center here.

File: docs/30DaysOfPWA/dev-tools/07.md [hunk 18]
```diff
@@ -15,3 +15,59 @@ Welcome to **Week 3 Day 7** of #30DaysOfPWA! Want to learn more about this proje
...
+After a few tests run to verify the quality of your PWA (which we all know is out of this world thanks to PWA Studio) you’ll receive a report card detailing what you do well and some areas for improvement.
+
+![pwa builder report card page](/docs/30DaysOfPWA/dev-tools/_media/07_pwabuilder_rc.png)
+ 
+At the bottom of this page, you can click next and press “Next” and you will be at the publishing page of PWABuilder. We can press “Store Package” in the Windows section and we are greeted with a pop up to input a few options.
+
+![pwa builder publish page](/docs/30DaysOfPWA/dev-tools/_media/07_pwabuilder_pub.png)
+   
+In a new tab, you are going to want to head over to Partner Center and register your app. 
+**It is important that you register your app on Partner Center with the same name you listed in your manifest. When it comes time to upload your app bundles, it will check if the bundle matches the name of the registered app.**
```
Comment (captainbrosset) [inline docs/30DaysOfPWA/dev-tools/07.md:55]: If you want this note to stand out even more, please add an empty line before and after. Without an empty line, the whole thing will be in the same paragraph with the line before, and the line after.

File: docs/30DaysOfPWA/dev-tools/07.md [hunk 19]
```diff
@@ -15,3 +15,59 @@ Welcome to **Week 3 Day 7** of #30DaysOfPWA! Want to learn more about this proje
...
+Information on how to register your app can be found here and information on where to find the necessary Ids can be found here. Once you have inputted your publisher information, you can press generate and download your app bundles. 
+
+### Step 2: Partner Center
+If you are inside of your app on Partner Center, you will see a button that says, “Start your submission”. After clicking on that you will see a list of options you will need to take care of before your submission is ready to be reviewed. Since you made sure to have everything on the checklist above, these sections should be a breeze. 
+
+![partner center submission page](/docs/30DaysOfPWA/dev-tools/_media/07_pc.png)
+ 
+After completing each task under the headers colored blue, your app is ready for review. The review process takes about 24 hours and the feedback will either be precisely what was wrong with your submission so you can fix it or a congratulations message letting your know that your submission has been processed and your app will be in the store within the next two hours.
```
Comment (captainbrosset) [inline docs/30DaysOfPWA/dev-tools/07.md:63]: Blue might not always be the colors that these headers are displayed with. And people using assistive technologies, or with low color vision will not experience this color. I think you can simply drop the "colored blue" part, and be more specific about which headers on Partner Center people are supposed to complete tasks under.

File: docs/30DaysOfPWA/dev-tools/07.md (hunk 6 above)
Comment (captainbrosset) [reply docs/30DaysOfPWA/dev-tools/07.md:70]: Agreed, let's keep it neutral, and only based on facts. We could also take this opportunity to mention some facts about other stores too.
I would also maybe move this section earlier in the article. I left a comment before saying that we should say that there are other stores, but in this article, we will focus on the Microsoft one. Right there, this section would make a lot of sense.

File: docs/30DaysOfPWA/dev-tools/07.md (hunk 8 above)
Comment (captainbrosset) [reply docs/30DaysOfPWA/dev-tools/07.md:73]: If we have the section about other stores earlier in the article, as suggested, then we don't really need the last sentence here about Google Play and Apple App Stores. 
Also, this part "your PWA will be hindered slightly in its capabilities" feels a bit out of place to me. I wouldn't want this blog series to make these kinds of statement about other stores without backing it up with plenty of evidence. So, either explain in details how the PWA would have less capabilities, or drop that part.

**Review by captainbrosset — CHANGES_REQUESTED** [review]
Great content to have in this blog series. It feels like we're really closing the loop here.
I made a number of comments to, hopefully, make the article a bit better.
Let me know what you think!

File: docs/30DaysOfPWA/dev-tools/06.md (hunk 1 above)
Comment (beth-panx) [reply docs/30DaysOfPWA/dev-tools/06.md:5]: Yes - this needs to be updated.

File: docs/30DaysOfPWA/dev-tools/07.md (hunk 4 above)
Comment (Jaylyn-Barbee) [reply docs/30DaysOfPWA/dev-tools/07.md:55]: yes! thanks for reminding me lol

File: docs/30DaysOfPWA/dev-tools/07.md [hunk 20]
```diff
@@ -13,5 +13,61 @@ Welcome to **Week 3 Day 7** of #30DaysOfPWA! Want to learn more about this proje
...
 | **Introduction** | Introduction to packaging for the Windows Store |
-| **The Process** | Overview of how to package for the actual process of using pwabuilder.com to package for the Windows Store |
+| **Step 1: Packaging** | A walkthrough of the PWA Builder process to package your app for the Windows Store |
+| **Step 2: Partner Center** | A walkthrough of Partner Center and how to submit your app to the Store |
 | **Why the Microsoft Store?** |An explanation of why you would chose the Microsoft Store over any others |
+
+# Packaging for the Windows Store
+
+## Introduction
+Great! You’ve just finished your web app and you are ready to send it off into the stores! Luckily for you, PWA Builder makes that as easy as pie. PWA Builder makes it possible to ship your app into multiple stores including the Google Play Store and Apple App Store. In this article we will cover how to use our tools to package your PWA for the Microsoft Store, via the Partner Center. This is the last step, sharing your PWA with the world and now its time for all your hard work to pay off! Before we get started, here are some things you might want to have ready:
```
Comment (beth-panx) [inline docs/30DaysOfPWA/dev-tools/07.md:23]:
```suggestion
Great! You’ve just finished your web app and you are ready to send it off into the stores! Luckily for you, PWA Builder makes that as easy as pie. PWA Builder makes it possible to ship your app into multiple stores including the Google Play Store and Apple App Store. In this article we will focus on how to use our tools to package your PWA for the Microsoft Store, via the Partner Center. This is the last step, sharing your PWA with the world and now its time for all your hard work to pay off! Before we get started, here are some things you might want to have ready:
```

**Review by beth-panx — APPROVED** [review]
@captainbrosset and @zateutsch - I worked with Jaylyn to address all your comments.
